training-Ex-2/print_word.py

```diff
@@ -0,0 +1,4 @@
+user='gnu-linux is rule the world now'
+for count_1 in user:
```

Comment:
give meaning full variable name. In this case instead of **count_1** you can use **letter**